### ISSUE TITLE
worker: add logging for queue operations

### ISSUE THREAD

**Issue by plyr4** [go-vela/community#588]
theres no logging around these queue operations
  https://github.com/go-vela/worker/blob/master/cmd/vela-worker/exec.go#L27-L35
  https://github.com/go-vela/worker/blob/master/cmd/vela-worker/operate.go#L77-L80

this would be useful for debugging platform issues.

**Comment by cognifloyd** [comment]
During development, I normally have Trace level logging enabled. With that, I see the pop call in exec():
https://github.com/go-vela/server/blob/master/queue/redis/pop.go#L18

That does not cover creating the new Queue in operate().

Should the log entry in Queue be changed to a different level, or add logging in the worker?
And what level should the new/changed logs be?

**Comment by plyr4** [comment]
right, I'm thinking of moving some of the logging over to INFO, as long as the context is attached.
im looking for consistency across info/trace logging
